apps/omg_performance/lib/omg_performance/sender_server.ex

```diff
@@ -144,6 +147,10 @@ defmodule OMG.Performance.SenderServer do
         _ = Logger.info("[#{inspect(seqnum)}]: Transaction submission will be retried, block is full.")
         :retry
 
+      {:error, {:client_error, %{"code" => "submit:utxo_not_found"}}} ->
+        _ = Logger.info("[#{inspect(seqnum)}]: Transaction submission will be retried, utxo not found.")
```

Comment:
why is this necessary now? Shouldn't the sender crash & terminate if `utxo_not_found`?

Reply:
Honestly I have no idea. I found out that:
1. [create_utxos_for_extended_perftest](https://github.com/omisego/elixir-omg/blob/c07675ba6c71799537e1b46965a31091838a4d04/apps/omg_performance/lib/performance.ex#L248) uses DepositHelper.deposit_to_child_chain/2
2. deposit_to_child_chain - waits until deposits are [recognized...](https://github.com/omisego/elixir-omg/blob/c07675ba6c71799537e1b46965a31091838a4d04/apps/omg/test/support/integration/deposit_helper.ex#L51)

But running the test as in demos with locally setup geth, ch-ch & watcher shows that `senders` starts before utxo appear in ch-ch.
I can create an issue to dig it further... (shall I?)

Reply:
I've never seen this when doing the older perf tests (simple/extended alike).

My biggest worry is that ignoring this error could lead to false positives in test runs of these tests done by the CI.

As a minimum I'd bump the log level to warn to make it annoy us and post an issue, if there's still an issue

Reply:
I resolved this (I think) in f3f4994911b6c16305bd484e2b200fa770676a01. The error handling is removed and tests pass